### DISCUSSION TITLE
Replace spatie/data-transfer-object for spatie/laravel-data

### DISCUSSION THREAD

**Discussion by yoelotero** [creagia/redsys-php#21] [Ideas]
spatie/data-transfer-object has stopped maintaining this package.

Their message:
Consider migrating to [spatie/laravel-data](https://spatie.be/docs/laravel-data) or [cuyz/valinor](https://github.com/cuyz/valinor).

**Comment by dtorras** [comment]
I would appreciate a PR to replace spatie/data-transfer-object. I've already looked at laravel-data, but is not a solution because it depends on Laravel

**Comment by dtorras** [comment]
Hi @yoelotero , I've replaced spatie/data-transfer-object in the last version v3.x
